### PR TITLE
Feature/arc 1110

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
https://meemoo.atlassian.net/browse/ARC-1110

Requires https://github.com/viaacode/hetarchief-proxy/pull/254

### CODE REVIEW

File: src/modules/media/types/index.ts [hunk 1]
```diff
@@ -5,7 +5,6 @@ export interface Media {
 	schemaIdentifier: string; // Unique id per object
 	meemooIdentifier: string; // PID (DON'T use this for identification of an object)
 	premisIdentifier: Record<string, string[]>;
-	premisRelationship: Record<string, string[]>;
 	premisIsPartOf?: string;
```
Comment (bertyhell) [inline src/modules/media/types/index.ts:8]: moeten we niet het veld premisIsPartOf wegnemen ipv premisRelationship?

Volgens de metadata set op confluence:
In de taak zeggen ze: alles onder relations wegnemen:
image

**Review by bertyhell — CHANGES_REQUESTED** [review]
see code remark

**Review by bertyhell — APPROVED** [review]
If charlotte signs off on this, it is good for me

**Comment by ian-emsens-shd** [comment]
> If charlotte signs off on this, it is good for me

https://meemoo.atlassian.net/browse/ARC-1109?focusedCommentId=34838
https://github.com/viaacode/hetarchief-proxy/pull/255

**Pull request merged**